crates/fuel-core/src/graphql_api/api_service.rs

```diff
@@ -287,6 +291,8 @@ where
         // `RequiredFuelBlockHeightExtension` uses the view set by the ViewExtension.
         // Do not reorder this line before adding the `ViewExtension`.
         .extension(RequiredFuelBlockHeightExtension::new())
+        .extension(CurrentStfVersionExtension::new())
+        .extension(CurrentConsensusParametersVersionExtension::new())
```

Comment:
I think spawning a separate extension for the field it is too overkill=D It decreases performance without any reason

Reply:
I think we can have one extension fully dedicated to return information to the end user. Like STFV, CPV, current block height(we can remove setting of the current height from the `RequiredFuelBlockHeightExtension`).

In this case we will have one extension that we can use to add any metadata

Reply:
The following changes have been made:
- Both STF and CP extensions have been merged into "chain state info" extension
- The "chain state info" extension is now also providing data about the current block height